Tella/Components/Connections/ConnectionsPageView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ConnectionsPageView.swift
+//  Tella
+//
+//  Created by Gustavo on 22/09/2023.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct ConnectionsPageView: View {
```

Comment:
I think the reference of this calss is still in the project if you can remove it